regression/contracts-dfcc/loop_contracts_memcmp/test.desc

```diff
@@ -0,0 +1,9 @@
+CORE gcc-only
```

Comment:
Could you elaborate as to why this test is GCC-specific?

Reply:
We use void pointer in this benchmark. Typechecking void pointer will trigger the error https://github.com/diffblue/cbmc/blob/develop/src/ansi-c/c_typecheck_expr.cpp#L4152 under VISUAL_STUDIO.

Reply:
Can we just use a char pointer instead?

Reply:
Actually, using char pointers will lead to a bug of DFCC. 
```
156: File: D:\a\cbmc\cbmc\src\goto-instrument\contracts\dynamic-frames\dfcc_instrument.cpp:330 function: dfcc_instrumentt::instrument_function
156: Condition: Precondition
156: Reason: found != goto_model.goto_functions.function_map.end()
156: << EXTRA DIAGNOSTICS >>
156: Function '_errno' must exist in the model.
156: << END EXTRA DIAGNOSTICS >>
```
There is a function `_errno` in the symbol table but not in the function map. I will keep this test as gcc-only and open an issue to track this bug.

Reply:
https://github.com/diffblue/cbmc/issues/7807